partner_contact_in_several_companies/models/ir_actions.py

```diff
@@ -21,3 +21,16 @@ def read(self, fields=None, context=None, load='_classic_read'):
                          "{'is_set': True, 'set_value': False},"),
                         1)
         return actions
+
+    @api.model
+    def _enable_context(self):
```

Comment:
This method is here only to make a test pass... please remove it.
It changes the context on a single action, where the `read` method changes it for every `res.partner` action.

The `read` method should be tested instead of modifying values in the database like this.
The test only checks one action, but all `res.partner` actions should have the same behaviour (even actions created on modules that are not in dependencies of this one, so changing data here is not the way to go).